src/ChronosDate.php

```diff
@@ -965,6 +1023,127 @@ public function isWeekend(): bool
         return in_array($this->dayOfWeek, Chronos::getWeekendDays(), true);
     }
 
+    /**
+     * Determines if the instance is yesterday
+     *
+     * @param \DateTimeZone|string|null $timezone Time zone to use for now.
+     * @return bool
+     */
+    public function isYesterday(DateTimeZone|string|null $timezone = null): bool
```

Comment:
These helpers now have a time zone parameter to allow comparing to the correct `now` but defaults to server time.